src/slave/containerizer/mesos/containerizer.cpp

```diff
@@ -363,6 +368,21 @@ Try<MesosContainerizer*> MesosContainerizer::create(
 
   Shared<Provisioner> provisioner = _provisioner->share();
 
+#ifdef __linux__
+  // Initialize either the cgroups v2 or cgroups v1 isolator, based on what
+  // is available on the host machine.
+  auto cgroupsIsolatorSelector = [] (const Flags& flags) -> Try<Isolator*> {
+  #ifdef ENABLE_CGROUPS_V2
+    Try<bool> supported =
+      mesos::internal::slave::cgroups2_isolator::supported();
+    if (supported.isSome() && *supported) {
+      return Cgroups2IsolatorProcess::create(flags);
+    }
+  #endif // ENABLE_CGROUPS_V2
+
+    return CgroupsIsolatorProcess::create(flags);
+  };
```

Comment:
Hm.. it seems like mounted is already sufficient? Ditto for the main code?

```suggestion
#ifdef __linux__
  // Initialize either the cgroups v2 or cgroups v1 isolator, based on what
  // is available on the host machine.
  auto cgroupsIsolatorSelector = [] (const Flags& flags) -> Try<Isolator*> {
#ifdef ENABLE_CGROUPS_V2
    Try<bool> mounted = cgroups2::mounted();
    if (mounted.isError() {
      return Error("Failed to determine if cgroups2 is mounted": + mounted.error());
    }
    if (*mounted) {
      return Cgroups2IsolatorProcess::create(flags);
    }
#endif // ENABLE_CGROUPS_V2

    return CgroupsIsolatorProcess::create(flags);
  };
```

Reply:
Removed this function altogether.